### DISCUSSION TITLE
TaskScheduler - Does it communicates over Internet

### DISCUSSION THREAD

**Discussion by harshvir** [dahall/TaskScheduler#977] [Q&A] [answered]
Hi,
We use the library TaskScheduler in our project and based on a customer request we are trying to figure out If the library communicates over to Internet any time during the course of operations?

It will be helpful if someone can confirm this.

**Comment by dahall** [comment, marked as answer]
It does no outbound communication internal to the library. It does provide an API for remote tasks to be created on other servers within a domain.
